### PR TITLE
Istio domainwide adminport

### PR DESCRIPTION
**Pull request by jshum2479**
This PR add support for operator 4.0 when istio is enabled, running against istio 1.10+

Test Results 
   https://build.weblogick8s.org:8443/view/all/job/weblogic-kubernetes-operator-kind-new/8209/

### CODE REVIEW

**Review by anpanigr — APPROVED** [review]
LGTM

File: operator/src/main/java/oracle/kubernetes/operator/helpers/PodStepContext.java [hunk 1]
```diff
@@ -1015,8 +1021,23 @@ private V1Probe createReadinessProbe(TuningParameters.PodTuning tuning) {
       boolean istioEnabled = getDomain().isIstioEnabled();
       if (istioEnabled) {
         int istioReadinessPort = getDomain().getIstioReadinessPort();
-        readinessProbe =
-            readinessProbe.httpGet(httpGetAction(READINESS_PATH, istioReadinessPort, false));
+        int localAdminPort = getLocalAdminProtocolChannelPort();
```
Comment (lennyphan) [inline operator/src/main/java/oracle/kubernetes/operator/helpers/PodStepContext.java:1024]: The variable `localAdminPort` doesn't look to be used?

**Review by lennyphan — APPROVED** [review]
Other than my comment, looks good.

File: operator/src/main/java/oracle/kubernetes/operator/helpers/PodStepContext.java (hunk 1 above)
Comment (jshum2479) [reply operator/src/main/java/oracle/kubernetes/operator/helpers/PodStepContext.java]: fixed

**Pull request merged**